sections/migrations.js

```diff
@@ -360,6 +360,7 @@ export default [
       "`tableName`: the table name used for storing the migration state (default `knex_migrations`)",
       "`schemaName`: the schema name used for storing the table with migration state (optional parameter, only works on DBs that support multiple schemas in a single DB, such as PostgreSQL)",
       "`disableTransactions`: don't run migrations inside transactions (default `false`)",
+      "`disableMigrationsListValidation`: don't validate that completed migrations are present in the appropriate directories (default `false`)",
```

Comment:
Could be expressed more specifically "do not validate that all the already executed migrations are still present in migration directories (default `false`)."

Reply:
Okay. updated